backend/core/src/listings/listings.service.ts

```diff
@@ -20,27 +22,34 @@ export class ListingsService {
       .leftJoinAndSelect("units.amiChart", "amiChart")
   }
 
-  public async list(jsonpath?: string): Promise<Listing[]> {
-    let listings = await this.getQueryBuilder()
-      .orderBy({
-        "listings.id": "DESC",
-        "units.max_occupancy": "ASC",
-        "preferences.ordinal": "ASC",
-      })
-      .getMany()
+  public async list(params: ListingsListQueryParams): Promise<Listing[]> {
+    const query = this.getQueryBuilder().orderBy({
+      "listings.id": "DESC",
+      "units.maxOccupancy": "ASC",
+      "preferences.ordinal": "ASC",
+    })
+
+    if (params.neighborhood) {
+      // This works because there's a 1:1 relationship between properties and listings.
```

Comment:
so one Property can be in many listings? So then the comment is probably not necessary at all?

Reply:
Yeah there can be multiple Listings associated with a single Property (I think in case some units in a building are e.g. available for seniors and others are available for general population, etc.). I think the comment is helpful to indicate that we can do this style of filtering for fields on Listing and Property, but not for fields on Unit.

I'd just change the comment to say that there is only one Property per Listing, instead of "a 1:1 relationship"

Reply:
Done!